### PR TITLE
Booker prize winners and nominees

### PR DESCRIPTION
**Pull request by sumanair**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.


**Describe the contribution to the taxonomy**

<!-- A concise description of what the contribution brings, replace "..." in the bullet list -->

- A knowledge contribution about winners and nominated authors of the Booker Prize


**Input given at the prompt**

<!-- What you entered, replace "..." -->

```
   Name five authors who have won multiple Booker Prizes.
```


**Response from the original model**


<!-- What you received from the original model in response to your input, 
replace "..." -->

```
   Here is a list of three books that have been shortlisted for the Man Booker Prize in 2017:                                                                                                                    
                                                                                                                                                                                                            
 - Lincoln in the Bardo by George Saunders                                                                                                                                                                     - Exit West by Mohsin Hamid                                                                                                                                                                                    - 4321 by Paul Auster                                                                                                                                                                                         
 You can read more about them at <https://www.theguardian.com/books/2017/sep/13/man-booker-prize-shortlist-announced-george-saunders-and-mohsin-hamid>.                  

```


**Response from the fine-tuned model**


<!-- Generate a synthetic dataset based on your newly added seed data; train the model 
with the synthetic data and now re-test the model's response with the same prompt.
Replace "..." with what you receive with the finetuned model. -->

```
J. M. Coetzee: two times for Life & Times of Michael K and Disgrace.
Peter Carey: two times for True History of the Kelly Gang and Oscar and Lucinda.
J. G. Farrell: two times for Troubles and The Siege of Krishnapur.
Hilary Mantel: two times for Wolf Hall and Bring Up the Bodies.
Margaret Atwood: two times for The Testaments and Alias Grace.


```

**Contribution checklist**

<!-- Insert an x between the empty brackets: [ ] >> [x] -->

- [x ] The contribution was tested with `ilab generate`
- [x ] No errors or warnings were produced by `ilab generate`
- [x ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ x] Content does not include PII or otherwise sensitive or confidential information
- [x ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by sumanair** [comment]
Hi all,
The DCO failure above seems to indicate that a commit was incorrectly signed off.  I remember signing, so i wanted to verify. This is what I see. Can someone please advise what corrective action to take? Thank you!
```
C:\Users\sumap\iLab>mkdir commit_check

C:\Users\sumap\iLab>cd commit_check

C:\Users\sumap\iLab\commit_check>git clone -b booker_prize https://github.com/sumanair/taxonomy.git
Cloning into 'taxonomy'...
remote: Enumerating objects: 2256, done.
remote: Counting objects: 100% (389/389), done.
remote: Compressing objects: 100% (188/188), done.
remote: Total 2256 (delta 276), reused 254 (delta 194), pack-reused 1867
Receiving objects: 100% (2256/2256), 1.84 MiB | 3.52 MiB/s, done.
Resolving deltas: 100% (843/843), done.

C:\Users\sumap\iLab\commit_check>cd taxonomy

C:\Users\sumap\iLab\commit_check\taxonomy>git branch
* booker_prize

C:\Users\sumap\iLab\commit_check\taxonomy>git log --pretty=format:"%h %s %b"
04e64f2 Booker prize winners and nominees Signed-off-by: sumanair <spnair@us.ibm.com>

7be4331 Merge branch 'instructlab:main' into booker_prize
3d05322 build(deps): bump actions/checkout from 4.1.6 to 4.1.7 (#1190) Bumps [actions/checkout](https://github.com/actions/checkout) from 4.1.6
to 4.1.7.

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
f625212 Knowledge Contribution: Man Booker Prize Winners and Nominees Signed-off-by: sumanair <spnair@us.ibm.com>

45b661c Knowledge Contribution: Man Booker Prize Winners and Nominees Signed-off-by: sumanair <spnair@us.ibm.com>

a18a0dc Merge branch 'instructlab:main' into main
78d3926 Update triaging-contributions.md (#1181) Added legal-hold label

Signed-off-by: JJ Asghar <awesome@ibm.com>
0bbfe0a Merge branch 'instructlab:main' into main
7520ecf build(deps): bump step-security/harden-runner from 2.8.0 to 2.8.1 (#1174) Bumps
[step-security/harden-runner](https://github.com/step-security/harden-runner)
from 2.8.0 to 2.8.1.

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
e7b1c33 Cleanup label workflow (#1171) Some tidying up in the label workflow.

---------

Signed-off-by: BJ Hargrave <hargrave@us.ibm.com>
a91b670 Fixes for the linter (#1166) These were created before the
- version: 2 was required
- attribution.txt was also required

Signed-off-by: JJ Asghar <awesome@ibm.com>
Co-authored-by: Mohit Gaur <56885276+Mohit-Gaur@users.noreply.github.com>
Co-authored-by: Anil Kumar Vishnoi <avishnoi@redhat.com>
Co-authored-by: Gerard Ryan <git@grdryn.xyz>
99b7781 Add skill for creating tables in Asciidoc format (#734) This is a re-submission of #187.

If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your
PR.


**Describe the contribution to the taxonomy**

<!-- A concise description of what the contribution brings, replace
"..." in the bullet list -->
```

**Comment by jjasghar** [comment]
I have fixed your DCO issue.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by sumanair** [comment]
> I have fixed your DCO issue.

Thanks @jjasghar How can I prevent this happening again? I signed my submissions, so I'm at a loss as to what caused it. Any advise?

**Comment by jjasghar** [comment]
I ended up having to rebase and squash your commits down to one. You had only one commit with the sign-off, which was casing the problem for the bot.

**Comment by sumanair** [comment]
> I ended up having to rebase and squash your commits down to one. You had only one commit with the sign-off, which was casing the problem for the bot.

But I had only one commit in that branch and i signed it as you can see from the query  dump I posted. strange. :) Does it look at other branches too? Do I need to do anything else in addition to adding the -s when I commit ? Perhaps I'm missing something...  Just trying to make sure this doesn't happen again.

**Comment by jjasghar** [comment]
Hi! 👋
It’s been a while since you’ve seen any movement on this PR. We haven’t forgotten about you!  We’ve run into some logistical issues, hence this delay. We absolutely want your PR, and being marketed as `e2e-ready` is still the last stop before we get it into the upstream model.

We are thankful for your patience and ask that you please keep this PR open. As soon as we finish all our behind-the-scenes work, we’ll test the full model against your submissions and, ideally, accept your amazing contribution(s)! 

Your Community Maintainer Team.

P.S. if you have any specific questions or thoughts, don’t hesitate to comment on pull request this or email [awesome@instructlab.ai](mailto:awesome@instructlab.ai) and [lhawthor@redhat.com](mailto:lhawthor@redhat.com), and we’ll get back to you as soon as possible.

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.
